observationtree/src/main/scala/explore/observationtree/ConstraintSetObsList.scala

```diff
@@ -407,36 +407,44 @@ object ConstraintSetObsList {
               // end of constraint set tree
               ReflexSplitter(propagate = true),
               // start of unassigned observations list
-              ReflexElement(size = 36, minSize = 36, clazz = ExploreStyles.ObsTreeSection)(
-                ReflexHandle()(
-                  Header(block = true,
-                         clazz = ExploreStyles.ObsTreeHeader |+| ExploreStyles.ObsTreeGroupHeader
-                  )(
-                    <.span(ExploreStyles.TargetLabelTitle)("UNASSIGNED OBSERVATIONS"),
-                    <.span(ExploreStyles.ObsCount, s"${unassignedObs.length} Obs")
-                  )
-                ),
-                Droppable(UnassignedObsId) { case (provided, snapshot) =>
-                  <.div(
-                    provided.innerRef,
-                    provided.droppableProps,
-                    props.getListStyle(snapshot.isDraggingOver)
-                  )(
-                    <.div(ExploreStyles.ObsTree)(
-                      <.div(ExploreStyles.ObsScrollTree) {
-                        Segment(
-                          vertical = true,
-                          clazz = ExploreStyles.ObsTreeGroup
+              ReflexElement(size = 36,
+                            minSize = 36,
+                            clazz = ExploreStyles.ObsTreeSection,
+                            withHandle = true
+              )(
+                ReflexWithHandle(reflexProvided =>
+                  Droppable(UnassignedObsId) { case (provided, snapshot) =>
+                    <.div(
+                      ExploreStyles.ObsUnassigned,
+                      provided.innerRef,
+                      provided.droppableProps,
+                      props.getListStyle(snapshot.isDraggingOver)
+                    )(
+                      ReflexHandle(provided = reflexProvided)(
+                        Header(block = true,
+                               clazz =
+                                 ExploreStyles.ObsTreeHeader |+| ExploreStyles.ObsTreeGroupHeader
                         )(
-                          unassignedObs.zipWithIndex.toTagMod(
-                            (props.renderObsBadgeItem _).tupled
-                          ),
-                          provided.placeholder
+                          <.span(ExploreStyles.TargetLabelTitle)("UNASSIGNED OBSERVATIONS"),
```

Comment:
I may rather not use capitals here and if we need them we set the style via css

Reply:
Good idea. I will do that.